index.js

```diff
@@ -0,0 +1,51 @@
+function solveEquation(ans) {
+  switch (ans[1]) {
+    case '+':
+      return ans[0] + ans[2]
+    case '-':
+      return ans[0] - ans[2]
+    case '*':
+      return ans[0] * ans[2]
+    case '/':
+      return ans[0] / ans[2]
```

Comment:
You should be typecasting these to numbers. Right now `'2'` would stay a string and `'2' + 2` is `'22'` not `4`.